tests/console/suseconnect_scc.pm

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2014-2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Do the registration against SCC after installation
+# Maintainer: Yi Xu <yxu@suse.de>
+
+use strict;
+use base "y2logsstep";
+
+use testapi;
+use registration;
+
+sub run {
+    return if get_var("HDD_SCC_REGISTERED");
+    my $version  = get_required_var("VERSION");
+    my $arch     = get_required_var("ARCH");
+    my $reg_code = get_required_var("SCC_REGCODE");
+    my $scc_url  = get_var("SCC_URL");
+    my $modules  = qw(base sdk desktop legacy script serverapp);
+
+    select_console 'root-console';
+    assert_script_run "SUSEConnect -r $reg_code";
+    assert_script_run "SUSEConnect --list-extensions";
+    assert_script_run "SUSEConnect --url $scc_url -r $reg_code" if $scc_url;
```

Comment:
I don't recommend to register first in L34 and register again here. Or do you have a specific reason to do that? I suggested to only append to the parameters based on $scc_url but only call the actual registration once

Reply:
Thanks for your suggestion. L34 and L35 are used for testing. I can remove them later.